### PR TITLE
[webui] Studio express workflow improvements

### PR DESCRIPTION
**Pull request by eduardoj**
Improve the workflow of building images:

- Add edit action for Kiwi Editor.
- Add the new layout for the overview page.
- Change the tabs for a better Image workflow experience and add the content of the editor as the new view for the edit action.
- Move detailed info of Image editor merging the info in the base_info partial and moving the dialog for editing to the description_form partial.
- Present in the view of kiwi image only the build results of the images, and not other repositories.
- Add version field to Kiwi image to the views and the model.

This pull request is based on a branch created by @mdeniz.

Pairprogrammed with @DavidKang, and with @mdeniz.

Here you have some screenshots of how this looks like:

screenshot from 2017-11-08 13-34-46_1_software
screenshot from 2017-11-08 13-35-19_2_details
screenshot from 2017-11-08 13-35-52_3_overview

### CODE REVIEW

**Comment by eduardoj** [comment]
[(8) P2: Studio Express Workflow](https://trello.com/c/mKEqUi23/607-8-p2-studio-express-workflow)

File: src/api/app/controllers/webui/kiwi/images_controller.rb [hunk 1]
```diff
@@ -27,33 +27,58 @@ def import_from_package
           end
         end
 
-        redirect_to kiwi_image_path(package.kiwi_image)
+        redirect_to edit_kiwi_image_path(package.kiwi_image, section: 'software')
```
Comment (hennevogel) [inline src/api/app/controllers/webui/kiwi/images_controller.rb]: 💭  why to edit?

**Comment by hennevogel** [comment]
@eduardoj if you approve code smells on codeclimate then please note down a reason next time...

**Comment by hennevogel** [comment]
As we are going to review this again next week together, let's get this in before...

**Comment by eduardoj** [comment]
@hennevogel:

> @eduardoj if you approve code smells on codeclimate then please note down a reason next time...

Ok.

File: src/api/app/controllers/webui/kiwi/images_controller.rb (hunk 1 above)
Comment (mdeniz) [reply src/api/app/controllers/webui/kiwi/images_controller.rb]: It was the show action before (and it was for showing/editing the info)... it  make more sense to edit in the edit action 😸

**Pull request merged**